Source/SwiftLintFramework/Rules/ColonRule.swift

```diff
@@ -15,6 +15,16 @@ public struct ColonRule: CorrectableRule, ConfigurationProviderRule {
 
     public init() {}
 
+    public var flexibleRightSpacing = false
+
+    public init(configuration: AnyObject) throws {
+        self.flexibleRightSpacing = configuration["flexible_right_spacing"] as? Int == 1
+    }
+
+    public var configurationDescription: String {
+        return "flexible_right_spacing: \(self.flexibleRightSpacing)"
```

Comment:
we avoid the use of `self.` when it can be omitted: https://github.com/github/swift-style-guide#only-explicitly-refer-to-self-when-required